sass/_ResponsiveTabs.scss

```diff
@@ -1,7 +1,8 @@
 .coveo-small-tabs {
 
-  .coveo-tab-dropdown-header {
+  #coveo-tab-dropdown-header {
```

Comment:
Why does this need to be an ID ?

If we don't need an ID, I'd prefer a class (we use classname everywhere else).